docs/content/guide/global-validators.md

```diff
@@ -163,7 +163,7 @@ export default {
   data() {
     const schema = {
       email: 'required|email',
-      password: 'required|min:8',
+      password: 'required|minLength:8',
```

Comment:
I don't think this is a mistake, any reason for the change?

Reply:
Because snippet description says

> Here is an example that uses all the rules we've defined prior in this page in a form validation schema:

`minLength` rule was defined in the previous section, so I guess it is a legit change

Reply:
You are correct, my bad. Thank you!